contributing.md

```diff
@@ -6,3 +6,22 @@ Contact the Videx team for any enquiries or questions you may have at this point
 
 * **Videx Team Lead**: Maja Irina Wegmann (mweg@equinor.com)
 * **Videx Tech Lead**: Kjerand Pedersen (kjerp@equinor.com)
+
+## Contributing
+
+In general, we follow the "fork-and-pull" Git workflow.
+
+1. Fork the repo on GitHub
+2. Clone the project to your own machine
+3. Work on your fork
+    1. Make your changes and additions
+        - Most of your changes should be focused on `src/` and `test/` folders and/or `README.md`. 
```

Comment:
should also include a line about adding an example if necessary under `examples/src`